README.md

```diff
@@ -187,8 +187,9 @@ representing the images encoded into TFRecords.
 Using the command line:
 
 ```bash
-tfrecorder check-tfrecords \
-    --file_pattern=/path/to/tfrecords/train*.tfrecord.gz \
+tfrecorder inspect \
+    --tfrecord-dir=/path/to/tfrecords/ \
+    --split='TRAIN' \
```

Comment:
I like this much better!

Reply:
Cool! I was also thinking we could maybe change the default behavior to dump some data for each split in a future PR.